src/core.lisp

```diff
@@ -1,3 +1,46 @@
 ;;;; Rasterization renderer.
 
 (in-package #:cl-raster/core)
+
+(ql:quickload :3d-vectors)
+(use-package :3d-vectors)
+
+(defun project-point-to-camera (camera point)
+  (let* ((to-point-vector (v- (camera-get-center camera) point))
```

Comment:
`camera` should probably be a class or a structure. Structure would define accessor `camera-center`, with class you would probably define an accessor `center`. Accessors are `setf`'able, meaning that you can set a field with `setf`:
```lisp
(setf (camera-center camera) new-center)
; or
(setf (center camera) new-center)
```
So the `get` part of the name is out of place.

Lastly, `camera` class should be part of the `#:cl-raster/scene` package. A local nickname `#:scene` is defined for core package, so you can use `(scene:center camera)` (or `(scene:camera-center camera)`)

You will need to export the corresponding symbol from `#:cl-raster/scene` package:
```lisp
  (:export #:center)  ; or #:camera-center
```

A simple definition of a structure would be
```lisp
(defstruct camera
  width
  height
  center
  direction)
```

Or a class:
```lisp

(defclass camera ()
  ((width :initarg :width :accessor width)
   (height :initarg :height :accessor height)
   (center :initarg :center :accessor center)
   (direction :initarg :direction :accessor direction)))
```